src/Data/Str.php

```diff
@@ -133,8 +133,8 @@ public static function hasLowerCase(string $str)
         $all_words = preg_split('/\s+/', $str);
 
         foreach ($all_words as $word) {
-            if (!preg_match_all('~^\p{Lu}~u', $word)) {
-                // There is one upper
+            if (!preg_match_all('~\p{Ll}~u', $word)) {
```

Comment:
The problem is about the `preg_match_all` return value.

The `preg_match_all` will return integer, I think we don't use the `!preg_match_all` to check whether the return value is.

It should use `preg_match_all(.....) === 0` condition.

Reference: https://www.php.net/manual/en/function.preg-match-all.php